Readme.md

```
@@ -1,3 +1,81 @@
+
+- **생애주기 중심 UX:** 사용자의 현재 상태에 맞는 다음 행동을 안내하는 역할별 운영 허브(마이페이지) 제공
+- **신뢰도 시스템:** 고용주의 솔직한 평가 점수와 이를 분석한 AI 리포트 제공
+- **안전한 프로세스:** 계약서 생성/조회/서명부터 S3 기반의 안전한 포트폴리오 파일 관리 및 법률 AI 가이드 연계
+
+---
+
+## 🛠 기술 스택 (Tech Stack)
+
+| 구분 | 기술 스택 |
+| :--- | :--- |
+| **Frontend** | <img src="https://img.shields.io/badge/Vue.js-4FC08D?style=flat&logo=vuedotjs&logoColor=white"/> <img src="https://img.shields.io/badge/TypeScript-3178C6?style=flat&logo=typescript&logoColor=white"/> <img src="https://img.shields.io/badge/Pinia-yellow?style=flat&logo=pinia&logoColor=white"/> <img src="https://img.shields.io/badge/Vite-646CFF?style=flat&logo=vite&logoColor=white"/> <img src="https://img.shields.io/badge/Tailwind_CSS-06B6D4?style=flat&logo=tailwindcss&logoColor=white"/> |
```

markdownlint-cli2 (0.21.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)